examples/door-lock-app/door-lock-common/door-lock-app.matter

```diff
@@ -360,6 +360,8 @@ server cluster DoorLock = 257 {
   readonly attribute int16u numberOfTotalUsersSupported = 17;
   readonly attribute int16u numberOfPINUsersSupported = 18;
   readonly attribute int16u numberOfRFIDUsersSupported = 19;
+  readonly attribute int8u numberOfWeekDaySchedulesSupportedPerUser = 20;
+  readonly attribute int8u numberOfYearDaySchedulesSupportedPerUser = 21;
   readonly attribute int8u maxPINCodeLength = 23;
```

Comment:
Why is NumberOfHolidaySchedulesSupported not enabled here?

Reply:
We had an internal discussion in the door-lock channel in slack and group decided that for V1 we include only weekday and yearday schedules. I think I can include holiday schedules if that is necessary or I can add it as a separate PR. Adding holiday schedule is a copy-paste pretty much anyway.

I should have left the comment about it, sorry.

Reply:
Separate PR is fine, but per spec if the SCH bit is set we have to include holiday schedules.

Maybe we should have separate feature bits here and a spec issue to make that change, so we can do just these two schedule types for V1?  Or make holiday schedules optional if SCH, not mandatory?

Reply:
I've created a spec issue so the group can decide. The issue number is [4882](https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/4882).